cmd/oras/root/cp.go

```diff
@@ -154,38 +154,45 @@ func doCopy(ctx context.Context, src oras.ReadOnlyGraphTarget, dst oras.GraphTar
 		return graph.Referrers(ctx, src, desc, "")
 	}
 
+	const (
+		Exists  = "Exists "
+		Copying = "Copying"
+		Copied  = "Copied "
+		Skipped = "Skipped"
```

Comment:
Do these variables need to be public? 
nit: Since these variables are passed to `PrintStatus()` as `status`, should we name them`statusExists`, `statusCopying`, etc.  for better readability?

Reply:
According to the naming of progress output, those should be `statusXXX`